src/containers/navbar-left/navbar-left.js

```diff
@@ -30,11 +25,16 @@ const NavbarLeft = () => {
     }
   };
 
-  const categoriesItems = categories.map(({ _id, name }) => (
-    <Link key={_id} className={styles.link} to={`/${getCategoryURL(name)}`}>
-      {name[LANGUAGE].value}
-    </Link>
-  ));
+  const CategoriesView = () => {
```

Comment:
Please, don't have two react components at the same file, it is confusing. If it is really big you can create for it a new folder and with standard component folder structure. If it is small and used only inside this component then, just revert these changes and add assign to the `categoriesItems` `null` if no categories have been fetched from the backend

Reply:
Fixed